Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
+ETW logs are available on CoreCLR only currently.
+
+
+# How to Collect ETW logs use logman
+
+
+## Start the logging
+
+ * Option 1: Generate a single file that does circular logging with Verbose level, For example
+ 
+ ```
+logman start WCFETWTracing -p "Microsoft-Windows-Application Server-Applications" 0xFFFFFFFF  0x5 -bs 64 -nb 120 320 -ets -ct perf -f bincirc -max 500 -o c:\temp\WCFEtwTrace.etl
```

Comment:
For end users, collect/analyze the server ETW traces is the same as any Desktop WCF services.

Only OSS users use bridge, which needs documentation as it does NOT enable typical desktop tracing experience.

The debugging experience for both users are different because of the bridge. I think it's easier for end users to read without mention bridge in the same document.